extension.json

```diff
@@ -220,6 +220,14 @@
                       },
                       "additionalProperties": false
                     },
+                    {
+                      "type": "object",
+                      "properties": {
+                        "standard": { "type": "string", "enum": ["Adobe"] },
+                        "version": { "type": "string" },
```

Comment:
This won't have a pattern you can verify against?

Reply:
This is a good idea, but unfortunately, it is too late to add validation here. We can't make old settings invalid now.